apps/app/src/views/routes/Settings/components/users/UserItem.tsx

```diff
@@ -42,6 +44,9 @@ export const UserItem = ({
       </Td>
       <Td>{user.email}</Td>
       <Td textOverflow={'ellipsis'}>{roles}</Td>
+      <Td>
```

Comment:
Looks like you need a corresponding header item (no title needed) 
<img width="182" alt="Screenshot 2024-01-04 at 9 52 06 AM" src="https://github.com/Photon-Health/client/assets/1777536/0e12b60e-5d53-44a3-8299-ee2bec1983d9">

Reply:
done